crates/driver/src/tests/cases/protocol_fees.rs

```diff
@@ -8,46 +8,66 @@ use crate::{
             ab_liquidity_quote,
             ab_order,
             ab_solution,
+            fee::{Policy, Quote},
             ExpectedOrderAmounts,
-            FeePolicy,
             Test,
         },
     },
 };
 
+struct Amounts {
+    sell: eth::U256,
+    buy: eth::U256,
+}
+
+struct Execution {
+    // The executed net-amounts (including network fee) reported by the solver
+    solver: Amounts,
+    // The executed net-amounts (including network and protocol) reported by the driver
+    driver: Amounts,
+}
+
+struct Order {
+    amounts: Amounts,
```

Comment:
supernanonit: flatten amounts to keep indentation minimal (little advantage in reusing amounts struct)